common/src/sql/dbinit.sql

```diff
@@ -370,6 +370,26 @@ CREATE INDEX ON omicron.public.disk (
 ) WHERE
     time_deleted IS NULL AND attach_instance_id IS NOT NULL;
 
+CREATE TABLE omicron.public.image (
+    /* Identity metadata (resource) */
+    id UUID PRIMARY KEY,
+    name STRING(63) NOT NULL,
+    description STRING(512) NOT NULL,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
+    /* Indicates that the object has been deleted */
+    time_deleted TIMESTAMPTZ,
+
+    project_id UUID,
+    volume_id UUID NOT NULL,
```

Comment:
Once the image has been constructed, it'll refer to a particular volume.

For the case of e.g. POST-ing an image by URL, would we immediately download + store the image as a part of invoking the POST request?

- If yes: Cool, this structure should work. We'll create a saga that does that work.
- If no: This structure may need to be more complex - we'll need to store the URL too.

Reply:
This is currently an unsolved problem, though that answer will likely be the same as others: some block migration task will move blocks from the read only parent to the volume region set(s), then remove the read only parent later. We can't gate the instance boot or image creation on that download though.

Reply:
Okay. I'll leave this question open as an implementation detail for follow-up PRs